### ISSUE TITLE
Remaining design

### ISSUE THREAD

**Issue by rowo** [OpenSRP/opensrp-client-reveal#17]
- [ ] Campaign and operational area filtering - https://marvelapp.com/project/3387341/screen/48317853/comments/
- [ ] Legend for items on map. This currently doesn't exist. My assumption is this is covered in training and isn't a problem since it hasn't been brought up. Is it in scope @craigappl ?
- [x] UI when a user needs to enable their location services when they hit the add point form and the my location button. I assume this should be the same thing Google Maps does. https://github.com/onaio/kujaku/issues/188

**Comment by craigappl** [comment]
@rowo that's correct, we do not need to post a legend. It's not in the current mSpray app.
